### ISSUE TITLE
input "fail_level" is not defined in action false positive

### ISSUE THREAD

**Issue by satmandu** [rhysd/actionlint#492]
I see this:
```
.github/workflows/Rubocop.yml:21:11: input "fail_level" is not defined in action "reviewdog/action-rubocop@v2". available inputs are "fail_on_error", "filter_mode", "github_token", "level", "only_changed", "reporter", "reviewdog_flags", "rubocop_extensions", "rubocop_flags", "rubocop_version", "skip_install", "tool_name", "use_bundler", "workdir" [action]
.github/workflows/ShellCheck.yml:19:11: input "fail_level" is not defined in action "reviewdog/action-shellcheck@v1". available inputs are "check_all_files_with_shebangs", "exclude", "fail_on_error", "filter_mode", "github_token", "level", "path", "pattern", "reporter", "reviewdog_flags", "shellcheck_flags" [action]
```
from https://github.com/chromebrew/chromebrew/blob/master/.github/workflows/Rubocop.yml and https://github.com/chromebrew/chromebrew/blob/master/.github/workflows/ShellCheck.yml but this error is a false positive.

https://github.com/reviewdog/action-rubocop/tree/v2 does support `fail_level`.

https://github.com/reviewdog/action-shellcheck/tree/v1 supports `fail_level` as well.

**Comment by rhysd** [comment]
This is already fixed on the HEAD of main branch. Please wait for the next release.
